db/python/layers/web.py

```diff
@@ -443,7 +450,7 @@ async def get_project_summary(
                 pmodels.append(
                     NestedParticipantInternal(
                         id=p['id'],
-                        external_id=p['external_id'],
+                        external_ids=json.loads(p['external_ids']),
```

Comment:
From database call here?

Reply:
I just followed the existing code in that file, which uses `json.loads` directly.

However — see other json-related reply — I think the way to go will be to change **all** the existing `from_db()` functions to use the new `from_db_json()` as appropriate, which I've got done modulo the question in the other reply.

Reply:
Yep, this sounds good to me!

Reply:
Excellent. It's now a bit off-topic for this PR, so I've separated it into PR #821, to come back to after we've merged this.